lms/static/js/views/image_field.js

```diff
@@ -70,7 +71,7 @@
             },
 
             showErrorMessage: function (message) {
-                return message;
+                throw('showErrorMessage not implemented for error: ' + message);
```

Comment:
What handles these exceptions?  How do you make sure this doesn't cause an issue that wasn't there?